20190301-Anforderungen/20190301-Anforderungen.tex

```diff
@@ -106,10 +106,12 @@ \subsubsection{Anforderungen an Lösungsansätze, Proof of Concept und MVP}
 	\item Das Proof of Concept muss technisch aufzeigen, wie viele RFID Tags in einer Sekunde gelesen werden können.
 	\item Das Proof of Concept soll eine RFID Lesezuverlässigkeit von 95\% aufweisen.
 	
-	\item Das MVP soll mit der Datenbank des Lagersystems kommunizieren können.
-	\item Das MVP soll in einem von Störfaktoren bereinigten Zustand die gleiche Anzahl RFID Tags lesen können wie im Proof of Concept definiert.
-	\item Das MVP soll erkennen, wenn eine Box ein Exemplar enthält, welches nicht dieser Box zugehörig ist und dies als eine Unstimmigkeit markieren.
-	\item Das MVP soll in der Lage sein, dem Endbenutzer in beliebiger Form mitzuteilen (kann Akustisch, Visuell oder durch Ausschleusen gelöst werden), welcher Behälter eine Unstimmigkeit enthält.
+	\item Das MVP soll mit einer Oracle Datenbank, mit einem Auszug des Datenbestandes des Lagerverwaltungssystems, oder direkt mit der Oracle Datenbank des Systems kommunizieren können.
+	\item Das MVP soll erkennen, wenn eine Box ein Exemplar (eines, welches mit RFID ausgestattet ist und technisch auch Lesbar ist) enthält, welches nicht dieser Box zugehörig ist.
+	\item Das MVP soll jede Unstimmigkeit (Exemplar, welches nicht zu diesem Behälter gehört) in einem Logdokument persistieren.
```

Comment:
in einem Logdokument ~persistieren~ festhalten.
Persistieren ist ein bisschen sehr technisch =)

Reply:
aber akurater und es sind ja technischi ahforderige XD

Reply:
Chani demit lebe 😄